ReClass.NET/Core/CoreFunctionsManager.cs

```diff
@@ -12,9 +12,10 @@ namespace ReClassNET.Core
 {
 	public class CoreFunctionsManager : IDisposable
 	{
-		private readonly Dictionary<string, ICoreProcessFunctions> functionsRegistry = new Dictionary<string, ICoreProcessFunctions>();
+		public readonly Dictionary<string, ICoreProcessFunctions> functionsRegistry = new Dictionary<string, ICoreProcessFunctions>();
 
 		public IEnumerable<string> FunctionProviders => functionsRegistry.Keys;
+        public ICoreProcessFunctions CurrentFunctions => currentFunctions;
```

Comment:
Could you please use tabs as indentation like the rest of the code?

Reply:
i think my ide uses spaces for some reason, can fix